Building on paths with spaces produces a near-empty RNG file

When building the MEI source using `teitorelaxng`, a path to the MEI source with a space in it will cause the script to output a nearly empty file.

To reproduce:

-- `cd` to a checkout of the MEI source with a space in it, e.g., `/home/joeblow/music encoding`
-- create an output directory, e.g., `build`
-- run the teitorelaxng script: `teitorelaxng --localsource=source/driver.xml customizations/mei-all.xml build/mei-all.rng`

You should get a notice:

```
using /home/joeblow/music encoding/trunk/source/driver.xml as default source
Convert mei-all.xml to /home/joeblow/music encoding/trunk/build/mei-all.rng (tei to relaxng) using profile default

BUILD FAILED
Target "encoding/trunk/source/driver.xml" does not exist in the project "teitorelaxng".

Total time: 0 seconds
```

A partial fix for this is to place quotes around the path on line 177 of teitorelaxng (note the swapping of the single and double quotes):

```
localsource=' -DdefaultSource="$REALSOURCE" '
```

This will cause the script to exit successfully; however, the schema file generated in the `build` directory is only 661 bytes long, and contains an (essentially) empty RNG schema.

If you rename the directory to one without spaces, and revert the change on line 177, everything will work.

I have not been able to verify if this is the same with a TEI build.

--- a/bin/transformtei
+++ b/bin/transformtei
@@ -174,7 +174,7 @@ else
   SDIR=`(cd $REALD; pwd)`
   REALSOURCE=$SDIR/`basename $TEISOURCE`
   echo using $REALSOURCE as default source
-  localsource=' -DdefaultSource="$REALSOURCE" '
+  localsource=" -DdefaultSource=\"$REALSOURCE\" "
 fi
 
 in=${1:?"no input file supplied; for usage syntax $0 --help"}

